src/Traits/PayPalRequest.php

```diff
@@ -120,13 +120,19 @@ private function setConfig(array $config = [])
      *
      * @return void
      */
-    protected function setClient()
+    protected function setClient($certificate = '')
     {
-        $this->client = new HttpClient([
-            'curl' => [
+	$curlConfig = [
                 CURLOPT_SSLVERSION     => CURL_SSLVERSION_TLSv1_2,
                 CURLOPT_SSL_VERIFYPEER => $this->validateSSL,
-            ],
+	];
```

Comment:
I would simply create a `$curlConfig`  variable inside this trait, so it is easily accessible as `$this->curlConfig`